scripts/basic-workflow-tests.sh

```diff
@@ -210,13 +230,13 @@ test_taxonomy() {
 
 test_generate() {
     task Generate synthetic data
-    if [ "$GRANITE" -eq 1 ]; then
-        GENERATE_ARGS+=("--model" "${DATA_HOME}/instructlab/models/granite-7b-lab-Q4_K_M.gguf")
-    elif [ "$MIXTRAL" -eq 1 ]; then
-        GENERATE_ARGS+=("--model" "${DATA_HOME}/instructlab/models/mixtral-8x7b-instruct-v0.1.Q4_K_M.gguf")
-    elif [ "$BACKEND" = "vllm" ]; then
-        GENERATE_ARGS+=("--model" "${DATA_HOME}/instructlab/models/instructlab/merlinite-7b-lab")
+    GENERATE_ARGS+=("--endpoint-url" "http://localhost:8000/v1")
```

Comment:
Why is this needed? Isn't the code doing this by default already? I mean it should look into the `serve` section and find this default, right?

Reply:
If `--endpoint-url` is not specified, `ilab data generate` will start up a temp server in the background (https://github.com/instructlab/instructlab/blob/main/src/instructlab/data/generate.py#L196). 

In the future we might want this behavior in this test, but for now we're directly serving the teacher model in the test.